tests/Capgemini.PowerApps.PackageDeployerTemplate.UnitTests/TemplateConfigTests.cs

```diff
@@ -159,6 +159,30 @@ public void Load_ActivateDeactivateSlasPopulated_ActivateDeactivateSlasDeseriali
             this.config.ActivateDeactivateSLAs.Should().BeTrue();
         }
 
+        [Fact]
+        public void Load_Tables_Exist()
+        {
+            this.config.Tables.Should().Contain(t => t.Name == "account");
+            this.config.Tables.Should().Contain(t => t.Name == "contact");
+        }
+
+        [Fact]
+        public void Load_Tables_Autonumber_Columns_Populated()
```

Comment:
For the test names I've been following Microsoft's suggestions here - https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests

`Method_Scenario_ExpectedBehaviour`

```suggestion
        public void Load_TableColumnAutoNumberSeedPresent_IsDeserialised()
```